requirements.txt

```diff
@@ -18,4 +18,36 @@ torch>=1.7.0
 torchvision>=0.8.1
 tqdm>=4.41.0
 
-pycocotools>=2.0
\ No newline at end of file
+pycocotools>=2.0
+
+# FROM BOX DETECTECTION: DUPLICATES POSSIBLE
+
+# base ----------------------------------------
+Cython
```

Comment:
There are duplicates in this file, (probably) won't cause an issue though. Also I'm not 100% sure if the comments could affect something